client.go

```diff
@@ -349,12 +354,15 @@ func (c *Client) doPaginatedRequest(req *http.Request, emptyResponse bool) (inte
 	}
 
 	responsePaginated := &Response{}
+	var curPage int
+
 	err = json.Unmarshal(responseBytes, responsePaginated)
 	if err == nil && len(responsePaginated.Values) > 0 {
 		var values []interface{}
 		for {
+			curPage++
 			values = append(values, responsePaginated.Values...)
-			if c.DisableAutoPaging || len(responsePaginated.Next) == 0 {
+			if c.DisableAutoPaging || len(responsePaginated.Next) == 0 || curPage >= c.LimitPages && c.LimitPages != 0 {
```

Comment:
This is still correct as a code, but it may be easier to read if it is wrapped in paren.

```suggestion
			if c.DisableAutoPaging || len(responsePaginated.Next) == 0 || (curPage >= c.LimitPages && c.LimitPages != 0) {
```

Reply:
nice idea. sure!
done ✅